version: '3.0'

scanning-settings:
  max-depth: 6
  exclude: []
  include-files: []
  system:
    targets: []


report:
  dependency-vulnerabilities:
    enabled: true
    auto-ignore-in-report:
      python:
        environment-results: true
        unpinned-requirements: true
      cvss-severity: []
      vulnerabilities:
        48545: # https://pyup.io/v/48545/97c [GDAL]
          reason: it's unclear whether we're vulnerable to this, see MAGIC/ops-data-store#29 for details
          expires: '2025-07-04'
        51832: # https://pyup.io/v/51832/97c [GDAL]
          reason: We won't be using GRIB2, see MAGIC/ops-data-store#29 for details
          expires: '2025-07-04'
        62283: # https://pyup.io/v/62283/97c [GDAL]
          reason: it's unclear whether we're vulnerable to this but we can't upgrade in any case
          expires: '2025-07-04'
        74054: # https://data.safetycli.com/v/74054/eda [GDAL]
          reason: not reviewed but we can't upgrade in any case
          expires: '2025-07-04'

fail-scan-with-exit-code:
  dependency-vulnerabilities:
    enabled: true
    fail-on-any-of:
      cvss-severity:
        - high
        - medium
        - critical
      exploitability:
        - high
        - medium
        - critical

security-updates:
  dependency-vulnerabilities:
    auto-security-updates-limit:
      - patch

